icon-registry/icons/action-filter_x16.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@rachelruderman It looks like this icon is missing some of the steps outlined in the instructions in the README file (e.g. `class="icon-dark"`). Probably worth having a look at the instructions before merging this one in. This process is kind of new to me as well, but I've been able to successfully merge some new icons in over the past couple weeks while Bart is out on paternity.